qemu/tests/win_virtio_driver_update_by_installer.py

```diff
@@ -37,14 +40,39 @@ def run(test, params, env):
     """
     def change_virtio_media(cdrom_virtio):
         """
-        change iso for virtio-win
+        change iso for virtio-win.
         :param cdrom_virtio: iso file
         """
         virtio_iso = utils_misc.get_path(data_dir.get_data_dir(),
                                          cdrom_virtio)
         test.log.info("Changing virtio iso image to '%s'", virtio_iso)
         vm.change_media("drive_virtio", virtio_iso)
 
+    def check_network_config(session_serial):
+        """
+        check ip and dns loss.
+        :param session_serial: session_serial
```

Comment:
You will need a blank line before this line according to the guidance.

Reply:
Hi @vivianQizhu, I have added a line already. Could you check again?